implementations/rust/ockam/ockam_core/src/lib.rs

```diff
@@ -51,12 +56,11 @@ pub use error::*;
 pub use message::*;
 pub use processor::*;
 pub use routing::*;
+#[cfg(feature = "std")]
```

Comment:
Could we avoid moving it? It makes sense to have println near the corresponding macro for no_std

Reply:
done